packages/tokamak/contracts-bedrock/scripts/verify-predeploy-contracts.sh

```diff
@@ -0,0 +1,345 @@
+             ["4200000000000000000000000000000000000042"]=""  # GovernanceToken : No Proxy
+             ["4200000000000000000000000000000000000000"]=""  # LegacyMessagePasser
+             ["4200000000000000000000000000000000000014"]=""  # L2ERC721Bridge
+             ["4200000000000000000000000000000000000017"]=""  # OptimismMintableERC721Factory
+             ["4200000000000000000000000000000000000018"]=""  # ProxyAdmin
+             ["4200000000000000000000000000000000000019"]=""  # BaseFeeVault
+             ["420000000000000000000000000000000000001a"]=""  # L1FeeVault
+             ["4200000000000000000000000000000000000020"]=""  # SchemaRegistry
+             ["4200000000000000000000000000000000000021"]=""  # EAS
+             ["4200000000000000000000000000000000000486"]=""  # ETH (TOKAMAK)
+)
+
+IMPLEMENTATION_SLOT="0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc"
+
+echo "Getting contracts addresses..."
+for address in ${!contracts[@]}; do
+  QUERY=.alloc.\"$address\".storage.\"$IMPLEMENTATION_SLOT\"
+  contracts[${address}]=$(echo $(curl -fsSL $GENESIS_URL | jq $QUERY) | cut -c 28-67)
+
+  if [ -z ${contracts[${address}]} ]; then
+    contracts[${address}]=${address}
+  fi
+done
+echo "Successfully getting contracts addresses!"
+
+# Path of contracts
+BASE_PATH=$(cd $(dirname $0)/.. && pwd -P)
+PROXY_PATH=${BASE_PATH}/src/universal/Proxy.sol
+L2_TO_L1_MESSAGE_PASSER_PATH=${BASE_PATH}/src/L2/L2ToL1MessagePasser.sol
+DEPLOYER_WHITE_LIST_PATH=${BASE_PATH}/src/legacy/DeployerWhitelist.sol
+WTON_PATH=${BASE_PATH}/vendor/WTON.sol
+L2_CROSS_DOMAIN_MESSENGER_PATH=${BASE_PATH}/src/L2/L2CrossDomainMessenger.sol
+L2_STANDARD_BRIDGE_PATH=${BASE_PATH}/src/L2/L2StandardBridge.sol
+SEQUENCER_FEE_VAULT_PATH=${BASE_PATH}/src/L2/SequencerFeeVault.sol
+OPTIMISM_MINTABLE_ERC20_FACTORY_PATH=${BASE_PATH}/src/universal/OptimismMintableERC20Factory.sol
+L1_BLOCK_NUMBER_PATH=${BASE_PATH}/src/legacy/L1BlockNumber.sol
+GAS_PRICE_ORACLE_PATH=${BASE_PATH}/src/L2/GasPriceOracle.sol
+L1_BLOCK_PATH=${BASE_PATH}/src/L2/L1Block.sol
+GOVERNANCE_TOKEN_PATH=${BASE_PATH}/src/governance/GovernanceToken.sol
+LEGACY_MESSAGE_PASSER_PATH=${BASE_PATH}/src/legacy/LegacyMessagePasser.sol
+L2_ERC721_BRIDGE_PATH=${BASE_PATH}/src/L2/L2ERC721Bridge.sol
+OPTIMISM_MINTABLE_ERC721_FACTORY_PATH=${BASE_PATH}/src/universal/OptimismMintableERC721Factory.sol
+PROXY_ADMIN_PATH=${BASE_PATH}/src/universal/ProxyAdmin.sol
+BASE_FEE_VAULT_PATH=${BASE_PATH}/src/L2/BaseFeeVault.sol
+L1_FEE_VAULT_PATH=${BASE_PATH}/src/L2/L1FeeVault.sol
+SCHEMA_REGISTRY_PATH=${BASE_PATH}/src/EAS/SchemaRegistry.sol
+EAS_PATH=${BASE_PATH}/src/EAS/EAS.sol
+ETH_PATH=
+
+function run() {
+  verify_proxy
+  verify_L2ToL1MessagePasser
+  verify_DeployerWhitelist
+  #verify_WTON
+  verify_L2CrossDomainMessenger
+  verify_L2StandardBridge
+  verify_SequencerFeeVault
+  verify_OptimismMintableERC20Factory
+  verify_L1BlockNumber
+  verify_GasPriceOracle
+  verify_L1Block
+  verify_GovernanceToken
```

Comment:
We don't use GoveranceToken contacts. So now we didn't deploy it In titan-sepolia-test.

Reply:
this mean is governanceToken?

Reply:
Yes. You can remove all of the GovernanceToken parts from this file.

Reply:
thank you! i fixed it